### ISSUE TITLE
Kastari menu is not in english

### ISSUE THREAD

**Issue by Immuuni** [jtiala/ruoka#4]
Hi! 

I noticed that the menu of Kastari is not available in english. At least, Aapo-application has also english menu so there should be suitable json somewhere?

**Comment by jtiala** [comment]
Yeah, it should be available. Uniresta menus are still a bit WIP.

**Comment by antarktikali** [comment]
The English menu can be obtained from the following url and parsed:
http://www.uniresta.fi/en/en-lunchlist.html?q=r35&_=1484133701716

The Uniresta website makes an ajax request to that url to get the English menu.

**Comment by jtiala** [comment]
@antarktikali Acctually I'm rewriting the back end for ruoka.kitchen at https://github.com/jtiala/ruoka-api, and I'm using that page there :)

**Comment by jtiala** [comment]
After it's ready (hopefully soon), I'll continue working on the front end using this repo.
